restli-tools/src/main/java/com/linkedin/restli/tools/clientgen/FluentApiGenerator.java

```diff
@@ -59,23 +55,8 @@
 {
```

Comment:
Since based on my understanding in the future their could be different implementations.

Right now it might be valuable (though not necessary) to mention current implementation is Parseq & Async based.

Same to the "TestGreetingsFluentClient".

Reply:
I think the implementation would be agnostic to the fluent client. Even if we change the implementation, it would just be a different set of templates. The generated API will not change (thus the Test class would remain unchanged)

Reply:
I know it is OK for now, just want to share that we can make a distinction now.

Because the tests in `TestGreetingsFluentClient` are not trulyi agnostic (you passed in Parseq component for the test.

I said this b ecause tests such as `TestActionResource` did make a distinction in the test name to suggest it is tests using ParSeq.

This is a suggestion and it will be your choice.

Reply:
agree with you, updated the test name.